config/components.php

```diff
@@ -138,11 +138,13 @@
 	 *
 	 * @param \Kirby\Cms\App $kirby Kirby instance
 	 * @param \Kirby\Cms\Collection $collection Collection of searchable models
-	 * @param string $query
-	 * @param mixed $params
-	 * @return \Kirby\Cms\Collection|bool
 	 */
-	'search' => function (App $kirby, Collection $collection, string $query = null, $params = []) {
+	'search' => function (
+		App $kirby,
+		Collection $collection,
+		string $query = null,
+		string|array $params = []
```

Comment:
Just to be sure, this change will not break existing search component installations, right?

Reply:
Changing the type hinting on the default component shouldn't change anything for custom components